linux/installer/common/gen_source/gen_source.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Well, the code in those scripts is python2, so it seems like a good idea to always make it use python2. I'll discard these.